src/components/editoptions/EditOptions.js

```diff
@@ -1,26 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { View, EditGray, Trash, ViewHidden } from '~/assets/image';
 
-const EditOptions = ({ isHidden, onHiddenIconClicked }) => {
+import { EDIT, DELETE, VIEW, VIEW_HIDDEN } from '~/components/icons/icon';
+
+const EditOptions = ({ isHidden = false, onHiddenIconClicked, onEditButtonClicked, onDeleteButtonClicked }) => {
   return (
-    <div className="edit-options">
-      <span className="edit-options__item" onClick={e => onHiddenIconClicked(e)}>
-        <img src={isHidden ? ViewHidden : View} alt="View" />
-      </span>
-      <span className="edit-options__item">
-        <img src={EditGray} alt="Edit" />
-      </span>
-      <span className="edit-options__item">
-        <img src={Trash} alt="Trash" />
-      </span>
-    </div>
+    <>
+      <div className="edit-options">
+        <span className="edit-options__item icon" onClick={onHiddenIconClicked}>
+          {isHidden ? VIEW_HIDDEN : VIEW}
+        </span>
+        <span className="edit-options__item icon" onClick={onEditButtonClicked}>
+          {EDIT('#B3B3B3')}
```

Comment:
instead of passing the hex value directly, you can add it to css and pass the css classname here.

Reply:
umm, can you give me hint on this?

Reply:
.color-1 {
color: '#B3B3B3'
}

Reply:
oo it is directly consuming the hex value. Then, you can place it in a constant.
const EDIT_ICON_COLOR = '#B3B3B3'

EDIT(EDIT_ICON_COLOR)

Reply:
done